apks/chromium/mk-vendor-file.py

```diff
@@ -13,7 +13,7 @@
 import subprocess
 import sys
 
-BASEDIR = "/mnt/cache/chromium"
+BASEDIR = "/tmp/cache/chromium"
```

Comment:
Please consider that tmp is usually <8GiB

Reply:
the nix build system already uses /tmp by default to do builds. if you have a ramdisk/tmpfs mounted to /tmp, you already can't build android (without a huge amount of ram). /mnt isn't writable by default from an unpriveleged user and running this script with sudo leads to unexpected results as the root user has different programs installed on nixos -- either a temp storage directory needs to be passed in somehow or we have to instruct users to make /mnt writable.

Reply:
My nix-daemon's TMPDIR is not /tmp/ for that very reason. /mnt is obviously bad but this just shouldn't be a decision the script makes itself.

A more sane default could be /var/tmp which is usually on a physical disk.

Reply:
to be clear, this isn't a build script that's called during a build. it's a script you run as an unprivileged user to create a new chromium manifest. so perhaps something under XDG_CACHE_HOME is most appropriate?

tbh I'm not sure why this script is so different from the ones that update the manifests for the main source dirs. it's much slower and significantly more brittle.